pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -393,76 +394,116 @@ func (r *rbac) createResources(ctx context.Context) error {
 		return err
 	}
 
+	// Semaphore to limit concurrency (max 100 goroutines at a time)
+	sem := make(chan struct{}, 100)
```

Comment:
we will need a configurable thing here, with a default of 50 for example

Reply:
This has been made configurable by introducing a new param in TektonConfig. 
![Screenshot From 2024-12-20 21-04-30](https://github.com/user-attachments/assets/69100b03-2ff2-4176-9352-76aea308a0a2)